### PR TITLE
Changing 302 to 301 for new IA redirects

### PR DESCRIPTION
**Pull request by matthewshaver**
## Prerelease docs
If this change is related to functionality in a prerelease version of dbt (delete if not applicable):
- [ ] I've added versioning components, as described in ["Versioning Docs"](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/versioningdocs.md)
- [ ] I've added a note to the prerelease version's [Migration Guide](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs/guides/migration/versions)

## Checklist
If you added new pages (delete if not applicable):
- [ ] The page has been added to `website/sidebars.js`
- [ ] The new page has a unique filename

If you removed existing pages (delete if not applicable):
- [ ] The page has been removed from `website/sidebars.js`
- [ ] An entry has been added to `_redirects`
- [ ] [Run link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

**Comment by matthewshaver** [comment]
@nghi-ly run tests passed in the environment

File: _redirects [hunk 1]
```diff
@@ -1,81 +1,81 @@
...
+/docs/building-a-dbt-project/snapshots /docs/build/snapshots 301
+/docs/building-a-dbt-project/seeds /docs/build/seeds 301
+/docs/building-a-dbt-project/building-models /docs/build/sql-models 301
+/docs/building-a-dbt-project/using-sources /docs/build/sources 301
+/docs/building-a-dbt-project/projects /docs/build/projects 301
+/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-upgrading-dbt-versions /docs/dbt-versions/upgrade-core-in-cloud 301
+/docs/core-versions /docs/dbt-versions/core 301
+/docs/dbt-cloud/cloud-dbt-cloud-support /docs/dbt-support 301
+/docs/about/viewpoint /docs/viewpoint 301
 /dbt-cli/configure-your-profile /docs/develop/connection-profiles 302
```
Comment (runleonarun) [inline _redirects]:
```suggestion
/dbt-cli/configure-your-profile /docs/develop/connection-profiles 301
```

File: _redirects [hunk 2]
```diff
@@ -1,81 +1,81 @@
...
 /docs/running-a-dbt-project/using-the-cli /docs/develop/about-the-cli 302
```
Comment (runleonarun) [inline _redirects:80]:
```suggestion
/docs/running-a-dbt-project/using-the-cli /docs/develop/about-the-cli 301
```

File: _redirects [hunk 3]
```diff
@@ -1,81 +1,81 @@
...
 /dbt-cli/install/from-source /docs/develop/source-install 302
```
Comment (runleonarun) [inline _redirects]:
```suggestion
/dbt-cli/install/from-source /docs/develop/source-install 301
```

**Review by runleonarun — APPROVED** [review]
@matthewshaver This looks good! There were a few we others we could change (anything with "develop" in the URL) which I suggested. I couldn't suggest fort these line but they can also be changed:

/dbt-cli/install/docker /docs/develop/docker-install 302
/dbt-cli/install/pip /docs/develop/pip-install 302
/dbt-cli/install/homebrew /docs/develop/homebrew-install 302
/dbt-cli/install/overview /docs/develop/installation 302
/docs/dbt-cloud/cloud-ide/the-dbt-ide /docs/develop/dbt-cloud-features 302'


Approving so you won't be help up!

**Pull request merged**